services/src/main/java/org/keycloak/services/resources/admin/ClientAttributeCertificateResource.java

```diff
@@ -145,8 +145,6 @@ public CertificateRepresentation generate() {
     @Tag(name = KeycloakOpenAPI.Admin.Tags.CLIENT_ATTRIBUTE_CERTIFICATE)
     @Operation( summary = "Upload certificate and eventually private key")
     public CertificateRepresentation uploadJks() throws IOException {
-        auth.clients().requireConfigure(client);
```

Comment:
Why remove this access check?

Reply:
The very next call is https://github.com/keycloak/keycloak/blob/669acefd5c42d880f203b9e59ef9cf536815766a/services/src/main/java/org/keycloak/services/resources/admin/ClientAttributeCertificateResource.java#L151 and the first thing what is called there is https://github.com/keycloak/keycloak/blob/669acefd5c42d880f203b9e59ef9cf536815766a/services/src/main/java/org/keycloak/services/resources/admin/ClientAttributeCertificateResource.java#L183-L184

so no point to check the permissions twice.